src/server/routes/group.js

```diff
@@ -35,20 +35,21 @@ module.exports = (aips) => {
     /**
      * GET groups for page
      */
-    router.get('/page/:categoryId(\\d+)/:num(\\d+)', asyncMiddleware(async(req, res) => {
+    router.get('/page/:categoryId(\\d+)/:num(\\d+)/:searchString', asyncMiddleware(async(req, res) => {
         const userId = req.session.user ? req.session.user.id : 0;
         const user = await User.findByPk(userId);
         const num = Number(req.params.num);
         const categoryId = Number(req.params.categoryId);
+        const searchString = String(req.params.searchString);
         if (num <= 0) {
             return res.status(404).send({ error: 'page not found' });
         }
-
         let groups = [];
         let groupCount = 1;
         const limit = 8;
         const offset = (num - 1) * limit;
-        if (categoryId === 0) {
+        
+        if (categoryId === 0 && searchString == "sentinelValue") {
```

Comment:
Instead of checking whether the string equals a specific string, how about just checking whether searchString is defined or not 
```js
if (categoryId === 0 && !searchString) {
```